kyc_account/models/account.py

```diff
@@ -15,7 +15,8 @@ def _onchange_partner_id(self):
 
     def _post(self, soft=True):
         if self.partner_id:
-            self.partner_id._kyc_accept_transaction(self)
+            for rec in self.partner_id:
+                rec._kyc_accept_transaction(self)
```

Comment:
This is not correct. The problem here is that self can be a recordset of account.move, it doesn't make sense to iterate `self.partner_id`. I suggest the following:

```python
def _post(self, soft=True):
    for rec in self:
        if rec.partner_id:
            rec.partner_id._kyc_accept_transaction(self)
     return super()._post(soft=soft)
```

Reply:
Here is problem with partner_id not with self actually